node/src/GlideClient.ts

```diff
@@ -172,7 +172,7 @@ export class GlideClient extends BaseClient {
      *   See https://redis.io/topics/Transactions/ for details on Redis Transactions.
      *
      * @param transaction - A Transaction object containing a list of commands to be executed.
-     * @param decoder - An optional parameter to decode all commands in the transaction. If not set, 'Decoder.String' will be used.
+     * @param decoder - An optional parameter to decode all commands in the transaction. If not set, the client decoder default config will be used.
```

Comment:
Copy updated doc here as well. Decoder should have the same doc all thoughout the client.